resources/qml/TimelineView.qml

```diff
@@ -252,7 +252,7 @@ Item {
             Layout.rightMargin: Nheko.paddingLarge
 
             TextArea {
-                text: TimelineManager.escapeEmoji(preview.roomTopic)
+                text: preview.roomName != "" ? TimelineManager.escapeEmoji(preview.roomTopic) : qsTr("This room is possibly inaccessible. If this room is private, you should remove it from the child list of this space.")
```

Comment:
Wouldn't it make more sense to just check for "isPreviewFetched" like you do below?

Reply:
The code below is a ``RoomlistModel``, while in TimelineView, ``room`` is a ``TimelineModel`` and ``preview`` is a ``RoomView``, neither of them have access to the ``isPreviewFetched`` property. Of course it should be possible to add it, but it seems overkill to me.

Reply:
I thought that rooms can't have empty names, and it seems I was wrong. There is a room in Matrix Community which doesn't have a name, but does have a topic. In the current state of Nheko, it displays like this:
![image](https://user-images.githubusercontent.com/25854770/226101305-f5a18cfa-c78d-485a-b3ae-52623981697c.png)
The "No preview available" part is wrong, since a preview *is* available. In this branch, the topic will therefore be replaced, which is even more wrong. I will add the ``isPreviewFetched`` property to either TimelineModel or RoomPreview to fix this.

Reply:
Fixed in https://github.com/Nheko-Reborn/nheko/pull/1404/commits/140736e109a913f2cc46646d672426348d91cf12

Reply:
Heh, I actually thought you were right, because when we have no name for a room, we generate one from the members. But we have no members in a preview, so of course we can't :D